oak_restricted_kernel/src/interrupts.rs

```diff
@@ -300,12 +294,12 @@ bitflags! {
     #[repr(transparent)]
     #[derive(Debug)]
     pub struct Mxcsr: u32 {
```

Comment:
I should really have taken a closer look; `x86_64::registers::mxcsr::MxCsr` _already is_ a bitflag, so printing it out would have done the right thing!

Sorry about that. Feel free to drop this wrapper, as the other bitflag is good enough.